contrib/localnet/bitcoin-sidecar/js/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "zeta-btc-client",
+  "version": "0.0.1",
+  "description": "The Zetachain BTC client",
```

Comment:
Let's not introduce nodejs for test cases but write them with existing go tooling.

Regarding update of `btcsuite/btcd` - I investigated this as as of now it's not trivial because we have a downstream deps of btcd v1:
- `zeta-chain/node`. Can be updated
  - `zeta-chain/go-tss` (thorchain's fork). Can be updated
    - `tss-lib` (thorchain). It uses `github.com/btcsuite/btcd` instead of `github.com/btcsuite/btcd/v2`

Reply:
So probably makes sense to cherry pick missing features from there and implement it natively using zeta's E2E suite.

cc @lumtis @ws4charlie

Reply:
Created issue for the improvement https://github.com/zeta-chain/node/issues/2900